.github/workflows/ci.yml

```diff
@@ -0,0 +1,84 @@
+---
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python CI
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  CI-Python:
+    runs-on: ubuntu-latest
+    env:
+      PYTHONDEVMODE: 1
+    strategy:
+      fail-fast: false
```

Comment:
What does it imply ? Don't we want to fail early in the setup to not push a half broken package if things break?
I'm not too familiar with this workflow yet! :)

Reply:
`fail-fast: false` implies that we won't cancel all workflows in the matrix, if one fails. Meaning if Python 3.9 fails, we don't stop the others. Can be useful to troubleshoot Python-specific issues faster.